pkg/ddevapp/commands.go

```diff
@@ -1,47 +1,69 @@
 package ddevapp
 
 import (
-	"github.com/ddev/ddev/pkg/fileutil"
-	"github.com/ddev/ddev/pkg/globalconfig"
-	"github.com/ddev/ddev/pkg/util"
-	copy2 "github.com/otiai10/copy"
+	"fmt"
 	"os"
+	"path"
 	"path/filepath"
+
+	dockerImages "github.com/ddev/ddev/pkg/docker"
+	"github.com/ddev/ddev/pkg/dockerutil"
+	"github.com/ddev/ddev/pkg/globalconfig"
+	"github.com/ddev/ddev/pkg/nodeps"
+	"github.com/ddev/ddev/pkg/util"
 )
 
-// PopulateCustomCommandFiles sets up the custom command files in the project
+// PopulateGlobalCustomCommandFiles sets up the custom command files in the project
 // directories where they need to go.
-func PopulateCustomCommandFiles(app *DdevApp) error {
-
+func PopulateGlobalCustomCommandFiles() error {
 	sourceGlobalCommandPath := filepath.Join(globalconfig.GetGlobalDdevDir(), "commands")
 	err := os.MkdirAll(sourceGlobalCommandPath, 0755)
 	if err != nil {
 		return nil
 	}
 
-	projectCommandPath := app.GetConfigPath("commands")
-	// Make sure our target global command directory is empty
-	copiedGlobalCommandPath := app.GetConfigPath(".global_commands")
-	err = os.MkdirAll(copiedGlobalCommandPath, 0755)
+	// Remove contents of the directory, if the directory exists and has some contents
+	commandDirInVolume := "/mnt/ddev-global-cache/global-commands/"
```

Comment:
This does seem like overkill, but same as what we're doing everywhere. It would be cool if we could do an rsync that would update+delete, but I don't see how to do that easily.

Reply:
Yeah I couldn't think of a way to do that - definitely worth investigating as a future enhancement though.